openmc/geometry.py

```diff
@@ -58,6 +58,25 @@ def __init__(
                     univ.add_cell(cell)
                 self._root_universe = univ
 
+    def __contains__(self, cell):
+
+        if cell is None:
+            return False
```

Comment:
I don't think this should be valid. It should raise a TypeError. This could be:

``` python
if not isinstance(cell, openmc.Cell):
   raise TypeError(...)
```

but I think there's check_value, which is the preferred way to do type enforcement.